### PR TITLE
Updated UI of Landing page node form.

### PR DESCRIPTION
**Pull request by sonnykt**
On-behalf-of: @salsadigitalauorg <sonny@salsadigital.com.au>

### CODE REVIEW

File: dev-tools.sh [hunk 1]
```diff
@@ -33,6 +33,6 @@
 # stable version.
 #
 # Uncomment and set the Dev-Tools's commit value and commit this change.
-# export GH_COMMIT=COMMIT_SHA
+export GH_COMMIT=079523e6d986c1acff871dffbb63a62cc66be685
```
Comment (sonnykt) [inline dev-tools.sh:36]: TODO: remove when https://github.com/dpc-sdp/dev-tools/pull/49 is merged

File: composer.json [hunk 2]
```diff
@@ -18,5 +18,14 @@
             "type": "composer",
             "url": "https://packages.drupal.org/8"
         }
+    },
+    "extra": {
+        "enable-patching": true,
+        "composer-exit-on-patch-failure": true,
+        "patches": {
+            "dpc-sdp/tide_core": {
+                "Moved Topic and Tags field to node form sidebar - https://github.com/dpc-sdp/tide_core/pull/213": "https://github.com/dpc-sdp/tide_core/pull/213.diff"
```
Comment (sonnykt) [inline composer.json:27]: TODO: to remove when https://github.com/dpc-sdp/tide_core/pull/213 is released.

File: composer.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
     "type": "drupal-module",
     "license": "GPL-2.0-or-later",
     "require": {
-        "dpc-sdp/tide_core": "^2.0.0",
+        "dpc-sdp/tide_core": "dev-develop as 2.0.0",
```
Comment (sonnykt) [inline composer.json:7]: TODO: to remove when https://github.com/dpc-sdp/tide_core/pull/213 is released.

File: dev-tools.sh [hunk 4]
```diff
@@ -33,6 +33,6 @@
 # stable version.
 #
 # Uncomment and set the Dev-Tools's commit value and commit this change.
-# export GH_COMMIT=COMMIT_SHA
+export GH_COMMIT=855db37db29f66885716bfe96461f32db12d7269
```
Comment (vincent-gao) [inline dev-tools.sh:36]: This should be reverted before merging.

**Review by vincent-gao — APPROVED** [review]

**Pull request merged**